vividus-plugin-datetime/src/main/java/org/vividus/expression/DiffDateExpressionProcessor.java

```diff
@@ -32,11 +32,15 @@
 import org.vividus.util.DateUtils;
 
 @Named
-public class DiffDateExpressionProcessor extends AbstractExpressionProcessor<String> implements NormalizingArguments
+public class DiffDateExpressionProcessor extends AbstractExpressionProcessor<String>
 {
-    private static final Pattern DIFF_DATE_PATTERN = Pattern
-            .compile("^diffDate\\((.+?),(?<!\\\\,)(.+?),(?<!\\\\,)(.+?),(?<!\\\\,)(.+?)(,(?<!\\\\,)(.+?))?\\)$",
+    // CHECKSTYLE:OFF
+    // @formatter:off
+    private static final Pattern DIFF_DATE_PATTERN = Pattern.compile(
+            "^diffDate\\((\\s*\"\"\".+?\"\"\"|.+?),(?<!\\\\,)(\\s*\"\"\".+?\"\"\"|.+?),(?<!\\\\,)(\\s*\"\"\".+?\"\"\"|.+?),(?<!\\\\,)(\\s*\"\"\".+?\"\"\"|.+?)(,(?<!\\\\,)(.+?))?\\)$",
```

Comment:
😵‍💫

Reply:
https://regex101.com/r/TJWYhc/1

Reply:
![image](https://user-images.githubusercontent.com/17816729/207288994-5a6054fd-f3e5-4f78-b0de-3dc26bbac413.png)

Reply:
have not found simpilier solution.